src/OrleansSQLUtils/CreateOrleansTables_MySql.sql

```diff
@@ -185,19 +184,13 @@ CREATE TABLE OrleansQuery
 		' 
 		INSERT INTO OrleansMembershipVersionTable
 		(
-			DeploymentId,
-			Version
+			DeploymentId
 		)
-		SELECT	
-			@deploymentId,
-			@version
-		WHERE NOT EXISTS
-		(			
-			SELECT 1
-			FROM OrleansMembershipVersionTable
-			WHERE DeploymentId = @deploymentId AND @deploymentId IS NOT NULL
-			FOR UPDATE
-		);
+		Values
+		(
+			@deploymentId
+		)
+		ON DUPLICATE KEY UPDATE DeploymentId = DeploymentId;
```

Comment:
More consideration from https://dev.mysql.com/doc/refman/5.7/en/insert-on-duplicate.html
> With ON DUPLICATE KEY UPDATE, the affected-rows value per row is 1 if the row is inserted as a new row, 2 if an existing row is updated, and 0 if an existing row is set to its current values. If you specify the CLIENT_FOUND_ROWS flag to mysql_real_connect() when connecting to mysqld, the affected-rows value is 1 (not 0) if an existing row is set to its current values.

Should one understand the above returns `0` in case of "no-op update", but only if all the values in the row are *set* to current values. Do they need to be set?